PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskTrackSkim.h

```diff
@@ -61,6 +61,8 @@ class AliAnalysisTaskTrackSkim : public AliAnalysisTaskEmcalJet
   // Configuration is handled via the YAML configuration file
   bool Initialize();
   void AddConfigurationFile(const std::string & configurationPath, const std::string & configName = "") { fYAMLConfig.AddConfiguration(configurationPath, configName); }
+  
+  void SetChargeScaling(bool chargeScaling);
```

Comment:
Please remove this setter and set the property via `RetrieveAndSetTaskPropertiesFromYAMLConfig()`. I prefer this approach since it gives me a separate file I can used to document settings